File: android/app/src/main/java/com/vegettable/app/model/DailyPrice.java
package com.vegettable.app.model;

import com.google.gson.annotations.SerializedName;

public class DailyPrice {
    @SerializedName("date")
    private String date;

    @SerializedName("avgPrice")
    private double avgPrice;

    @SerializedName("volume")
    private double volume;

    public String getDate() { return date; }
    public double getAvgPrice() { return avgPrice; }
    public double getVolume() { return volume; }
}

File: android/app/src/main/java/com/vegettable/app/model/DailyPrice.kt
package com.vegettable.app.model

import com.google.gson.annotations.SerializedName

class DailyPrice {
    @JvmField
    @SerializedName("date")
    val date: String? = null

    @JvmField
    @SerializedName("avgPrice")
    val avgPrice: Double = 0.0

    @JvmField
    @SerializedName("volume")
    val volume: Double = 0.0
}
